cvmfs/cvmfs.cc

```diff
@@ -2626,7 +2644,8 @@ static int AltProcessFlavor(int argc, char **argv) {
     return quota::MainCacheManager(argc, argv);
   }
   if (strcmp(argv[1], "__wpad__") == 0) {
-    return download::MainResolveProxyDescription(argc, argv);
+    return download::MainResolveProxyDescription(argc, argv,
+        cvmfs::statistics_);
```

Comment:
This is a special mode of operation.  The Statistics object should be created on the fly in `download::MainResolveProxyDescription()`